targets/p5.js

```diff
@@ -0,0 +1,188 @@
+import {
+  sculptToGLSL,
+  uniformsToGLSL,
+} from "../generators/sculpt.js";
+import {
+  usePBRHeader,
+  useHemisphereLight,
+  sculptureStarterCode,
+  fragFooter,
+} from "../glsl/glsl-lib.js";
+
+export function toGLSL(source) {
+  if (typeof source === "function") {
+    source = source.toString();
+    source = source.slice(source.indexOf("{") + 1, source.lastIndexOf("}"));
+  } else if (!(typeof source === "string")) {
+    throw "sculptToHydraRenderer requires the source code to be a function, or a string";
```

Comment:
I think we can change "Hyrda" -> to "p5"

Reply:
oops good catch! Also I forgot that this copy-and-pasted function already handles passing in a function as the source instead of a string, so I don't need to export two separate functions for this.